src/components/Filters.module.css

```diff
@@ -0,0 +1,43 @@
+@value colors: "../styles/colors.module.css";
+@value black, grey, white, orange, green, light-gray from colors;
+
+.filter {
+  padding: 2px 22px;
+  color: white;
+  background: gray;
+  transition: background 0.1s linear;
+  border: none;
+  margin-right: 10px;
+  border-radius: 20px;
+  position: relative;
+  cursor: pointer;
+}
+
+.filter:global(.active),
```

Comment:
What does `:global()` do? I read a few answers on Google and I want to make sure I fully understood this :joy: Does it force `.active` class to have a name without a "unique prefix identifier"? So, instead of having `class="filter_809hC active_809xD"` we have `class="filter_809hC active"`?

Also, is this a good practice or are we doing this to avoid any bug?

Plus, is it a CSS Module feature or is it a Webpack's CSS Loader feature? I got confused about it too, although I can find references to `:local` and `:scope` in both docs. :yum:

Reply:
1 - `So, instead of having class="filter_809hC active_809xD" we have class="filter_809hC active"?` yep, it works exactly like that!

2 - `Also, is this a good practice or are we doing this to avoid any bug?` actually you are right, in this case, we don't need to use `:global`, I've sent an adjustment. thanks for the great review!! 👊 

`Plus question` = CSS Modules is just a "spec" actually, and CSS loader implements it. that's why you can find references in both docs. I've read rumors but can't remember where that google chrome was looking to implement it as a native feature in browsers. 👀

Reply:
Ohhh, good to know that! I really didn't know CSS Modules was just a spec. So many things make sense now! :joy: Thank you, man! :rocket: :rocket: